### PR TITLE
Rpc calls visual fixes

### LINKED ISSUE #69: Couple of visual issues with RPC
screenshot from 2016-04-27 21 41 16

1./4. We should avoid having scroll if it's possible to break response into multiple lines.
2. Tooltip is not visible for the first item
3. Icons are overlaying text / content - we should have some separate space for them.

### PR DESCRIPTION
**Pull request by Adam-Goldman7**
resolves #69

### CODE REVIEW

File: client/components/RpcCalls/index.js [hunk 1]
```diff
@@ -63,12 +64,13 @@ export default class RpcCalls extends Component {
               <div className='col col-6'>
                 {this.renderForm()}
               </div>
-              <div className='col col-6'>
+              <div className='col col-6' onMouseLeave={() => this.setState({hoveredCallIdx: null})}>
                 {this.renderClear()}
                 <h2 className={styles.header}>History</h2>
-                <div className={`${styles.history} row`}>
+                <div className={`${styles.history} row`} id='styles-history'>
```
Comment (tomusdrw) [inline client/components/RpcCalls/index.js]: Please no ids!

File: client/components/RpcCalls/index.js [hunk 2]
```diff
@@ -248,10 +274,24 @@ export default class RpcCalls extends Component {
...
 
-  renderPrevCallsToolbar (call) {
+  renderPrevCallsToolbar () {
+    const idx = this.state.hoveredCallIdx;
+    if (typeof idx !== 'number') {
+      return;
+    }
+    const call = this.props.rpc.prevCalls[idx];
+    const callId = `call-${idx}`;
+    const wrapStyle = {top: document.getElementById(callId).offsetTop - 22};
```
Comment (tomusdrw) [inline client/components/RpcCalls/index.js]: Use `rel` for that - it's exactly what it was made for.

File: client/components/RpcCalls/index.js [hunk 3]
```diff
@@ -248,10 +274,24 @@ export default class RpcCalls extends Component {
...
+    if (hasScrollbar('styles-history')) {
+      wrapStyle.right = 13;
+    }
+
     return (
-      <div className={styles.callActionsWrap}>
-        <IconButton className={styles.callActionsButton}><MoreVertIcon /></IconButton>
+      <div
+        className={styles.callActionsWrap}
+        style={wrapStyle}
```
Comment (tomusdrw) [inline client/components/RpcCalls/index.js]: What do you try to achieve by this? We should avoid any DOM operations to compute that - there must be some smarter way of doing this.

Comment (Adam-Goldman7) [inline client/components/RpcCalls/index.js]: the action menu is not under the same dom element as the calls
this is to escape the overflow problem
so I'm computing the y position the action menu should have

**Pull request merged**